source/Generic/PlayState/PlayState.cs

```diff
@@ -120,6 +119,14 @@ public override void OnApplicationStarted(OnApplicationStartedEventArgs args)
                     .Show();
                 ProcessStarter.StartUrl(@"https://github.com/darklinkpower/PlayniteExtensionsCollection/wiki/PlayState#window-notification-style-configuration");
             }
+
+            var suspendPlaytimeOnlyFeature = PlayniteApi.Database.Features.FirstOrDefault(a => a.Name.Equals("[PlayState] Suspend Playtime only", StringComparison.OrdinalIgnoreCase));
```

Comment:
I'm not understanding, what is the purpose of changing the name of the feature?

Reply:
Because it was intended to be only one feature instead 2, since one of them will be pointless. If you have the setting enabled, the Suspend Playtime only feature will have no effect, and if you have it disabled the new feature that I would create will not have effect.

Reply:
This would cause issues and create confusion when you want `X` game to suspend playtime only for any reason, like having anticheat or example. Then if you change PlayState mode, it would do the opposite and it wouldn't be clear why.